### ISSUE TITLE
Axis numberFormatter #% does not round all labels

### ISSUE THREAD

**Issue by fundo-dev** [amcharts/amcharts4#1503]
With the following configuration:
```
    "xAxes": [
        {
            "type": "ValueAxis",
                "ticks": {
                    "disabled": false,
                    "length": 5,
                    "strokeOpacity": 1,
                    "stroke": "#999"
                }
            },
            "numberFormatter": {
                "type": "NumberFormatter",
                "numberFormat": "#%"
            }
        }
```
some numbers are not rounded.

![image](https://user-images.githubusercontent.com/6403296/62036852-e3967080-b1f2-11e9-9e30-fa7d08bd0515.png)

Example:
https://codepen.io/ivanmoretti/pen/Xvpzzp

**Comment by martynasma** [comment]
Gah, damn floating point bug again. Thanks!

Will let you know when a fix is available.

**Comment by martynasma** [comment]
This issue was fixed in dev version, and will be released soon.

We'll notify you here when that happens.

Meanwhile you can fix it by using the following format `"#.%"`.

**Comment by fundo-dev** [comment]
Thanks!
Shall I always use format `"#.%"` instead of `"#%"` if I don't want to display decimals?

**Comment by martynasma** [comment]
Yup. If you know you never want any decimals, then this format is the way to go.

Another option is to use `maxPrecision`.

https://www.amcharts.com/docs/v4/reference/valueaxis/#maxPrecision_property

**Comment by martynasma** [comment]
Fixed in [4.5.11](https://github.com/amcharts/amcharts4/releases/tag/4.5.11).

## [4.5.11] - 2019-07-31

### Added
- New `AxisDataItem` properties: `minPosition` and `maxPosition`. Enable controlling visibility of associated axis element (e.g. label, tick, grid). Overrides `minLabelPosition` and `maxLabelPosition`.

### Fixed
- `NumberFormatter` was sometimes susceptable to floating-point error when formatting percent formats (`#%`);
- Cursor hover was not working with shadow DOM.
- `DateFormatter` could sometimes parse string-based dates wrong when in UTC mode.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
